### PR TITLE
implemented task and tests

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
-# Boilerplate for layout tasks
+# Human rights
 
+Transform the preamble and the first 16 articles of the following document into 
+a web page: https://www.ohchr.org/EN/UDHR/Documents/UDHR_Translations/eng.pdf
+
+The font of the page must be `Arial`. Set the base font size to `13px`.
+
+The "Universal Declaration of Human Rights" must be a `h1` header, styled as
```
Comment (danmysak) [inline readme.md]: А можеш, будь ласка, замінити _header_ на _heading_? Вибач.

File: readme.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
 
+Transform the preamble and the first 16 articles of the following document into 
+a web page: https://www.ohchr.org/EN/UDHR/Documents/UDHR_Translations/eng.pdf
+
+The font of the page must be `Arial`. Set the base font size to `13px`.
+
+The "Universal Declaration of Human Rights" must be a `h1` header, styled as 
+`17px bold`.
+
+The "Preamble" and "Article N" must be `h2` headers, styled as `15px non-bold`.
```
Comment (danmysak) [inline readme.md]: headings

File: readme.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+
+The vertical spacing around "Preamble" and "Article N" must be `1em`. All other 
+vertical spacing must be set to zero, as in the document.
+
+Every paragraph should go into a `<p>` tag (the preamble is supposed to be a 
+single paragraph divided with `<br>` when necessary).
+
+Put numbered lists into `<ol>`s.
+
+After article 16 don’t display any more articles, but put an `h2` header reading
```
Comment (danmysak) [inline readme.md]: heading

File: config/backstop/backstopConfig.js [hunk 4]
```diff
@@ -35,7 +35,36 @@ const config = {
...
-    { ...basicScenario }
+    {
+      ...basicScenario,
+      label: 'entire-document',
+      selectors: ['document']
+    },
+    {
+      ...basicScenario,
+      label: 'h1-heading',
+      selectors: ['h1']
```
Comment (roman-mate) [inline config/backstop/backstopConfig.js]: are these selectors used to generate screenshots? 

I'd say this PR has too many screenshots. Let's reduce that to 2-3 screenshots?

Comment (ivadimko) [reply config/backstop/backstopConfig.js]: @roman-mate This is a specific task where we need to check how they write `HTML`. Basically, there are some `<p>` tags with `<br>` inside. We also have to make sure they use `h2` for level-2 headings.
Also splitting the entire document to small screenshots helps us to check even `1%` mismatch. 

If it is ok for you to remove `h2` and `p` from checking I can do it as well. This way we'll have 
  - one `document` screenshot  
  - one `h1` screenshot
  - one `a` screenshot

Comment (roman-mate) [reply config/backstop/backstopConfig.js]: yeah, that makes sense. Let's just write a selector that returns one element instead of multiple.

Comment (ivadimko) [reply config/backstop/backstopConfig.js]: for the moment we can't force them to write specific selectors. They have just empty `index.html` and they know nothing about `id`, `class`, `data-attributes` etc. So the best way is to keep only selectors which return a small number of screenshots (like `h1`)

Comment (roman-mate) [reply config/backstop/backstopConfig.js]: How about h1:nth-of-type(100n+1)?

Comment (ivadimko) [reply config/backstop/backstopConfig.js]: good idea, thank you

**Pull request merged**